README.md

```diff
@@ -1,6 +1,6 @@
 # TensorFlow Encrypted
 
-[![CircleCI Badge](https://circleci.com/gh/mortendahl/tf-encrypted/tree/master.svg?style=svg)](https://circleci.com/gh/mortendahl/tf-encrypted/tree/master) [![GitHub](https://img.shields.io/github/license/mortendahl/tf-encrypted.svg)](./LICENSE) [![PyPI](https://img.shields.io/pypi/v/tf-encrypted.svg)](https://pypi.org/project/tf-encrypted/)
+[![CircleCI Badge](https://circleci.com/gh/mortendahl/tf-encrypted/tree/master.svg?style=svg)](https://circleci.com/gh/mortendahl/tf-encrypted/tree/master) [![GitHub](https://img.shields.io/github/license/mortendahl/tf-encrypted.svg)](./LICENSE) [![PyPI](https://img.shields.io/pypi/v/tf-encrypted.svg)](https://pypi.org/project/tf-encrypted/) ![Status](https://img.shields.io/badge/status-alpha-blue.svg)
```

Comment:
Thinking we could organize these a little bit better for first-time users based on what they'd be most interested in. Perhaps put the status first, next with license, then the pypi, and finally the build?